src/mappings/constants.ts

```diff
@@ -17,6 +17,9 @@ export const EVENT_TYPES = {
   STORAGE: "storage",
   SUBMIT_PROPOSAL: "submit_proposal",
   TRANSFER: "transfer",
+  LIQUIDATING: "liquidating",
```

Comment:
EVENT_TYPES are used for different purposes. These are states of Vaults, so a new enum called VAULT_STATES will be better.